sdks/python/apache_beam/options/value_provider.py

```diff
@@ -101,6 +102,11 @@ def get(self):
   @classmethod
   def set_runtime_options(cls, pipeline_options):
     RuntimeValueProvider.runtime_options = pipeline_options
+    # Set is_time_counter_enabled = True if 'time_per_element_instrumentation'
```

Comment:
You can drop the comments here, it is exactly describing what the code does and does offer additional context to the reader.

Reply:
Deleted.